lib/ipam/ipam_types.go

```diff
@@ -117,3 +117,39 @@ type PoolUtilization struct {
 	// Utilization for each of this pool's blocks.
 	Blocks []BlockUtilization
 }
+
+// Consumer represent an entity consuming an IP address.
```

Comment:
I don't think these structs are used any more, so we can get rid of them.

Reply:
Oh, I see - they are still in the calicoctl PR.

I think we can move the definition of these into the calicoctl repo, at least, since that is the only place they are used. No need for them to be publicly defined here.

Reply:
removed from libcalico.